cpp/log/cert_submission_handler.h

```diff
@@ -29,9 +30,20 @@ class CertSubmissionHandler {
   util::Status ProcessPreCertSubmission(cert_trans::PreCertChain* chain,
                                         ct::LogEntry* entry) const;
 
-  // For clients, to reconstruct the bytestring under the signature
-  // from the observed chain. Does not check whether the entry
-  // has valid format (i.e., does not check length limits).
+  // For clients, to reconstruct all possible bytestrings under the signature
+  // from the observed chain. Does not check whether the entry has valid format
+  // (i.e., does not check length limits). Returns the number of entries
+  // written or an error meesage.
+  static util::StatusOr<size_t> X509ChainToEntries(
```

Comment:
This is just `entries->size()`, anyway? I'd just return a `util::Status`, in that case.

Reply:
done